pkg/reconciler/instances/istio/README.md

```diff
@@ -1,29 +1,31 @@
-# istio-configuration reconciler
+# Istio-configuration reconciler
 
 ## Overview
 
-The istio-configuration reconciler manages `istio-configuration` Kyma component.
+Istio-configuration reconciler manages the [istio-configuration](https://github.com/kyma-project/kyma/tree/main/resources/istio-configuration) Kyma component.
 
-## Run Reconciler locally (Mac OS)
+## Prerequisites
 
-Follow these steps to run istio-configuration reconciler locally on your cluster:
+The `istio-configuration` component requires `cluster-essentials` to be installed beforehand for Istio to be functioning properly.
```

Comment:
```suggestion
The Istio component requires `cluster-essentials` to be installed beforehand for Istio to be functioning properly.
```
+ note stating to check the Istio component name in specific version (i'm not sure if that's needed cause i don't plan any other component renaming)